modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,46 @@
+
+[abstract]
+{description}
+
+
+When you set the bucket property `enableCrossClusterVersioning` (ECCV) to `true`, for each document processed by XDCR, XDCR stores an internal history of the document in the extended attributes. This set of information is also called Hybrid Logical Vector (HLV), which is a set of Hybrid Logical Clock (HLC) information.
+
+NOTE: The bucket property `enableCrossClusterVersioning` must be set to `true` to use the `mobile=[Off | Active]` flag, and this flag in turn  enables the setting _XDCR Active-Active with Sync Gateway 4.0+_. For more information about XDCR Active-Active with Sync Gateway 4.0+, see xref:learn:clusters-and-availability/xdcr-active-active-sgw.adoc[XDCR Active-Active with Sync Gateway].
+
+[#hlv-data-maintained-in-xattr]
+== Hybrid Logical Vector (HLV) data maintained in the extended attributes
+
+The new metadata, HLV, is stored as a system user created extended attribute (xattr). The HLV metadata is also called Version Vectors. The system xattr name is `"_vv" -- "xattrs"."_vv"`. 
+
+The HLV metadata takes up `109 + 40N` bytes of space per document, where N is the number of buckets mutating the document in the replication topology. 
+
+NOTE: The size of the HLV metadata must remain constant as long as your replication topology is constant. However, the HLV data accumulation occurs when the replication topology changes and the document copy goes through different clusters, for example, when being restored to a new cluster.
+
+To remove the accumulated or outdated HLV data, the HLV metadata is pruned periodically. You can control the pruning frequency by setting the bucket property `versionPruningWindowHrs`.
+
+IMPORTANT: This is an important limitation to the _XDCR Active-Active with Sync Gateway_ feature. If you use the user created extended attributes (xattrs) in your documents, and you have more than 10 user xattrs in a document, then you cannot use the feature _XDCR Active-Active with Sync Gateway_. This is due to an internal limitation of managing extended attributes in a document. If you try to use the feature _XDCR Active-Active with Sync Gateway_ when you have more than 10 user xattrs in your document, the XDCR replication **silently skips** replicating the document that has more than 10 user xattrs. As a result, the source and target data in the replication skipped documents will not be consistent. The only way you will know this skip occured is because the Prometheus stat **subdoc_cmd_docs_skipped** will be incremented and the document will _not_ be consistent between the target and source. So, if your documents have more than 10 user xattrs, do _not_ enable the property `enableCrossClusterVersioning` with a purpose to use the _XDCR Active-Active with Sync Gateway_ feature.
```

Comment:
This IMPORTANT note should be in the section "Dependent XDCR features" under the paragraph about the mobile setting.

Currently, the mobile/XDCR feature information is mixed up with the HLV information.   So, the mobile/XDCR information should be moved to its own section called "Dependent XDCR features".